packages/design-system/src/Button/Button.styles.tsx

```diff
@@ -0,0 +1,246 @@
+  --button-color-border: var(--ads-v2-color-border);
+  --button-font-weight: 600;
+  --button-font-size: 14px;
+  --button-icon-size: 16px;
+  --button-padding: 8px 12px;
+`;
+
+const Sizes = {
+  sm: css`
+    --button-font-weight: 500;
+    --button-font-size: 12px;
+    --button-padding: 4px 8px;
+    --button-icon-size: 12px;
+  `,
+  md: css`
+    --button-font-weight: 600;
+    --button-font-size: 14px;
+    --button-padding: 8px 12px;
+    --button-icon-size: 16px;
+  `,
+  lg: css`
+    --button-font-weight: 600;
+    --button-font-size: 16px;
+    --button-padding: 8px 16px;
+    --button-icon-size: 18px;
+  `,
+};
+
+const Kinds = {
+  primary: css`
+    --button-color-bg: var(--ads-v2-color-bg-brand);
+    --button-color-fg: var(--ads-v2-color-fg-on-brand);
+    --button-color-border: var(--ads-v2-color-border-brand);
+
+    &:hover {
+      --button-color-bg: var(--ads-v2-color-bg-brand-emphasis);
+      --button-color-fg: var(--ads-v2-color-fg-on-brand);
+      --button-color-border: var(--ads-v2-color-border-brand-emphasis);
+    }
+
+    &:active {
+      --button-color-bg: var(--ads-v2-color-bg-brand-emphasis);
+      --button-color-fg: var(--ads-v2-color-fg-on-brand);
+      --button-color-border: var(--ads-v2-color-border-brand-emphasis);
+    }
+  `,
+  secondary: css`
+    --button-color-bg: var(--ads-v2-color-bg);
+    --button-color-fg: var(--ads-v2-color-fg-muted);
+    --button-color-border: var(--ads-v2-color-border);
+
+    &:hover {
+      --button-color-bg: var(--ads-v2-color-bg-subtle);
+      --button-color-fg: var(--ads-v2-color-fg-muted);
+      --button-color-border: var(--ads-v2-color-border);
+    }
+
+    &:active {
+      --button-color-bg: var(--ads-v2-color-bg-muted);
+      --button-color-fg: var(--ads-v2-color-fg-muted);
+      --button-color-border: var(--ads-v2-color-border-emphasis);
+    }
+  `,
+  tertiary: css`
+    --button-color-bg: transparent;
+    --button-color-fg: var(--ads-v2-color-fg);
+    --button-color-border: transparent;
+
+    &:hover {
+      --button-color-bg: var(--ads-v2-color-bg-subtle);
+      --button-color-fg: var(--ads-v2-color-fg-muted);
+    }
+
+    &:active {
+      --button-color-bg: var(--ads-v2-color-bg-muted);
+      --button-color-fg: var(--ads-v2-color-fg-muted);
+    }
+
+    &:disabled {
+      --button-color-bg: transparent;
+    }
+  `,
+  error: css`
+    --button-color-bg: var(--ads-v2-color-bg-error);
+    --button-color-fg: var(--ads-v2-color-fg-on-error);
+    --button-color-border: transparent;
+
+    &:hover {
+      --button-color-bg: var(--ads-v2-color-bg-error-emphasis);
+      --button-color-fg: var(--ads-v2-color-fg-on-error-emphasis);
+      --button-color-border: transparent;
+    }
+
+    &:active {
+      --button-color-bg: var(--ads-v2-color-bg-error-emphasis-plus);
+      --button-color-fg: var(--ads-v2-color-fg-on-error-emphasis);
+      --button-color-border: transparent;
+    }
+  `,
+};
+
+const LoaderSizes = {
```

Comment:
Ideally this should be part of the spinner styles.

Reply:
Thank you for pointing out. I will change this